packages/paste-website/src/pages/layout/stack/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Stack - Layout
+description: Stack is a horizontal and vertical flow component used to build layouts.
```

Comment:
I would say something more along the lines of creating space between components, rather than building layouts.

Reply:
I made that change. Let me know if it works.